service-python/pdfgenerator/src/lib/queues.py

```diff
@@ -21,17 +20,21 @@ def on_generate_callback(channel, method, properties, body):
 
         # binding_key = method.routing_key
         message = json.loads(body)
-        # logging.info(f" [x] {binding_key}: Received message: {message}")
+        # logging.info(f" [x] Received message: {message}")
```

Comment:
why comment out? either remove or add back.

Reply:
These logging statements can sometimes be useful during debugging and integration with other services but they should be commented out or removed for any deployment.